ush/setup.py

```diff
@@ -1219,16 +1308,18 @@ def get_location(xcs,fmt):
     # experiment directory (EXPTDIR).
     #
     if not RUN_TASK_MAKE_GRID:
-        if (GRID_DIR is None):
+        if GRID_DIR is None:
             GRID_DIR = os.path.join(DOMAIN_PREGEN_BASEDIR, PREDEF_GRID_NAME)
 
-            msg = dedent(f"""
+            msg = dedent(
+                f"""
                GRID_DIR not specified!
                Setting GRID_DIR = {GRID_DIR}
-            """)
+            """
+            )
             logger.warning(msg)
 
-        if not os.path.exists(GRID_DIR): 
+        if not os.path.exists(GRID_DIR):
             raise FileNotFoundError(
                 f'''
```

Comment:
For consistency, should the single quotes be converted to double quotes, then convert the double quotes around `{GRID_DIR}\` to single quotes?